docs/tools/piral-inspector.md

```diff
@@ -4,4 +4,215 @@ title: Piral Inspector
+
+On the tools bar, there now will also appear the ```Piral``` icon, and with a click on it, you will see the initial "Not connected" page. 
+
+In case you are currently running a Piral instance in debug mode, you will see the available Pilets and registered routes.
+
+![GoogleChromeExtensionToolPiralInspector](../diagrams/piralinspector_chrome_add-on_tool.png)
+
+
+### Opera
+
+Type the following URL into the browser address line followed by hitting enter:
+```
+addons.opera.com/extensions
+```
+
+On the following page, you can ```search for "Piral Inspector"```, and you will see the add-on as the first result.
+
+Click on ```"Add to Opera"```.
+ 
+![OperaExtensionAddToOperaPiralInspector](../diagrams/piralinspector_opera_add-on_searchresult.png)
+
+Now, a security message will appear, please click on ```"Add extension"``` to accept the required security settings:
+
+![OperaExtensionSecurityMessagePiralInspector](../diagrams/piralinspector_chrome_add-on_security.png)
+
+Piral Inspector is now available, and you can open it via the Developer Tools (```Developer > Developer Tools```).
+
+On the tools bar, there now will also appear the ```Piral``` icon, and with a click on it, you will see the initial "Not connected" page. 
+
+In case you are currently running a Piral instance in debug mode, you will see the available Pilets and registered routes.
+
+![OperaExtensionToolPiralInspector](../diagrams/piralinspector_chrome_add-on_tool.png)
+
+
+## Using Piral Inspector
+
+Piral Inspector offers a wide variety of capabilities regarding Piral instance and its corresponding Pilets.
+
+The add-on/extension operates on a ```locally in debug mode running Piral instance```, and it directly interacts with it.
+
+The following sections are shown inside the extension:
+- Information about Piral instance
+- Available Pilets
+- Add Pilets
+- Registered Routes
+- Events
+- State Container
+
+All of these sections will documented in the following chapters:
+
+### Piral instance
+In the very left top, the extension shows the name and the version of the currently running Piral instance.
+Besides that, in the very right top, a click on the icon allows the user to adapt the extension's settings to its needs.
+
+![PiralInstanceToolPiralInspector](../diagrams/piralinspector_tool_piralinstance.png)
+
+In the debug setting, the user can activate/deactivate the following options:
+![DebugSettingsToolPiralInspector](../diagrams/piralinspector_tool_debugsettings.png)
+
+#### State Container Logging
+Activates/Deactivates the logging of changes in the state container to the Developer Tools console.
+
+#### Load available Pilets
+Activates/Deactivates the loading of the available Pilets.
+> This option is only available when the user is debugging a Pilet.
+
+#### Full refresh on change
+Refreshes the current browser page in case there was a change in the debugged Pilet.
+> This option is only available when the user is debugging a Pilet.
```

Comment:
Instead of a quote, consider using a box. I would use the warning: https://docs.piral.io/reference/documentation/test

```md
::: warning: Only for Pilets
This feature has an impact only when the emulator is used, i.e., when `pilet debug` is running.
```

Reply:
Done. Thanks for the hint.